frontend/src/metabase/entities/collections/getExpandedCollectionsById.js

```diff
@@ -28,7 +28,6 @@ function getExpandedCollectionsById(
           : null,
       parent: null,
       children: [],
-      is_personal: c.personal_owner_id != null,
```

Comment:
Now BE is sending `is_personal` to every collection in `GET /api/collections` so, FE don't need to do this anymore.